util/dvsim/Testplan.py

```diff
@@ -293,7 +318,7 @@ def __init__(self, filename, repo_top=None, name=None):
                 "progress": 0.0,
             }
 
-    def _parse_testplan(self, filename, repo_top=None):
+    def _parse_testplan(self, filename, repo_top=None, tags=None):
```

Comment:
As with `_create_testplan_elements`, I probably wouldn't include a default value for the argument (or for `repo_top`, for that matter).